Makefile

```diff
@@ -18,38 +12,15 @@ bootstrap:
 	@echo Nothing to do.
 
 docker-build:
+	curl -Ls https://dl.bintray.com/deis/deisci/objstorage-0314fc5-linux-amd64 -O
+	mv objstorage-0314fc5-linux-amd64 rootfs/bin/objstorage
+	chmod +x rootfs/bin/objstorage
```

Comment:
@kmala can you do this download step inside the `Dockerfile`? Something like this should work:

```
ADD https://dl.bintray.com/deis/deisci/objstorage-0314fc5-linux-amd64 /bin/objstorage
```